docs/MS Excel/Границы ячеек.md

```diff
@@ -0,0 +1,28 @@
+≡ Краткое руководство Markdown
+
+# Границы ячеек
+
+**Рисование границ**
```

Comment:
[ch-md-error-emphasis-instead-of-a-heading](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-emphasis-instead-of-a-heading)